packages/ui/components/form/inputs/Input.tsx

```diff
@@ -123,6 +123,9 @@ export const InputField = forwardRef<HTMLInputElement, InputFieldProps>(function
           {...labelProps}
           className={classNames(labelClassName, labelSrOnly && "sr-only", props.error && "text-error")}>
           {label}
+          {!readOnly && passThrough.required ? (
```

Comment:
So that InputField can show the * in itself if it's a required field. Should check other usages of InputField to ensure that there are no more than 1 star(in case the user of InputField was adding * itself).

Reply:
Thanks @CarinaWolli . It was unintentional. Though I feel that it should be there in App Input Fields too, I have removed it from everywhere except Booking Page(as that's a separate discussion)